### PR TITLE
Link updates and logo update

### PR DESCRIPTION
**Pull request by Zoobdude**

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -46,7 +46,6 @@ const roles = {
 const channels = {
   HelpDesk: '1049703391762321418',
   General: '994044917355663450',
-  BotSpam: '1159083520027787307',
```
Comment (danieldietzler) [inline src/constants.ts:49]: This is still used

**Review by danieldietzler — CHANGES_REQUESTED** [review]

File: src/constants.ts (hunk 1 above)
Comment (danieldietzler) [reply src/constants.ts:49]: It's just a dev-only channel

Comment (Zoobdude) [reply src/constants.ts:49]: ahh, I see

**Review by danieldietzler — APPROVED** [review]
Thank you! :)

**Review by bo0tzz — APPROVED** [review]

**Pull request merged**